validmind/models/foundation.py

```diff
@@ -59,9 +58,7 @@ def predict(self, X: pd.DataFrame):
         """
         Predict method for the model. This is a wrapper around the model's
         """
-        return np.array(
-            [self.predict_fn(self._build_prompt(x[1])) for x in X.iterrows()]
-        )
+        return [self.predict_fn(self._build_prompt(x[1])) for x in X.iterrows()]
```

Comment:
We didn't need to convert into numpy array since the `assign_predictions()` is already doing it.

Reply:
👍